BuiltInCommands.cpp

```diff
@@ -81,4 +80,48 @@ void GetCurrDirCommand::execute()
 {
     this->getOutputStream() << SmallShell::getInstance().getWorkingDir() << std::endl;
 }
+#pragma endregion
+
+#pragma region JobsCommand
+
+#pragma endregion
+
+#pragma region ForegroundCommand
+ForegroundCommand::ForegroundCommand(const char *cmd_line,JobsList *jobs) : BuiltInCommand(cmd_line)
+{
+    this->jobs = jobs;
+}
+void ForegroundCommand::execute()
+{
+    JobsList::JobEntry *job = nullptr;
+    // check if there is any id with the command
+    if (cmd_v.size() > 1)

```

Comment:
Please review this condition, it should probably be 2 @boazfrider 
```suggestion
    if (cmd_v.size() > 2)

```